src/std/text.ab

```diff
@@ -376,11 +378,8 @@ pub fun reversed(text: Text): Text {
 /// }
 /// ```
 pub fun starts_with(text: Text, prefix: Text): Bool {
-  let result = trust $ if [[ "{text}" == "{prefix}"* ]]; then
-    echo 1
-  fi $
-
-  return result == "1"
+    trust $ [[ "{text}" == "{prefix}"* ]] $
```

Comment:
I'm not sure how you do it in Amber, but as far as I understand the cheapest check possible will look something like `test "{text}" == "{prefix}"*`

I mean we probably don't want to check for the exit code, we want just return the result of test

Reply:
Ah, I see! Probably still don't need to extract the status before returning it?